CorsixTH/Lua/entities/humanoids/staff/receptionist.lua

```diff
@@ -84,7 +84,10 @@ end
 
 -- Helper function to decide if Staff fulfills a criterion
 -- (one of "Doctor", "Nurse", "Psychiatrist", "Surgeon", "Researcher" and "Handyman")
-function Receptionist:fulfillsCriterion(criterion) -- luacheck: no unused args
+function Receptionist:fulfillsCriterion(criterion)
+  if criterion == "Receptionist" then
+    return true
+  end
   return false
```

Comment:
The entire body could be `return criterion == "Receptionist"`

Reply:
Kept it in line with Handyman and Nurse instances, where it does it the long way too.

Doco wise I thought I would check fulfillsCriterion was ok as its used for checking staff for the room. As that same comment is on all the derived classes where this is overridden also and then it has Junior/Consultant in the Doctor case, which aren't criterion for a room anymore, and maybe they need to be separated in another method.

Reply:
We have had quite a few starting programmers, and often they don't know that "==" delivers a boolean value. Simple beginner programs don't manipulate boolean values so they never see such things. For this reason starting programmers think that comparisons exclusively belong in an `if` or `while`, and booleans exist as `true` and `false` only. And then you get `if x==y then return true else return false end` as result.
As we have had quite a few starting programmers, there are lots of such instances in the code.

As for documentation, I am fine if you change allowed input or change output or change aim of the code, but please reflect that in changes of the documentation.